src/services/expenses.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-console */
+'use strict';
+
+let expenses = [];
+
+const reset = () => {
+  expenses = [];
+};
+
+const filterExpenses = ({ userId, categories, from, to }) => {
+  if (userId) {
+    expenses = expenses
+      .filter((expense) => expense.userId === +userId);
+  }
+
+  if (categories) {
+    expenses = expenses
+      .filter((expense) => categories.includes(expense.category));
+  }
+
+  if (from && to) {
+    expenses = expenses.filter(({ spentAt }) => (
+      spentAt >= from && spentAt <= to
+    ));
+  }
+
+  return expenses;
+};
+
+const getExpenseById = (expenseId) => {
+  const foundExpense = expenses.find((expense) => expense.id === +expenseId);
+
+  return foundExpense || null;
+};
+
+const createExpense = (data) => {
+  const newExpense = {
+    id: Math.random(expenses.length + 1),
```

Comment:
Use a more robust method for generating IDs, like UUID, to avoid potential collisions when creating new expenses.
Or make a separate function for find max id & plus 1 to this value